webview-ui/src/components/AnalysisPage.tsx

```diff
@@ -154,12 +165,25 @@ const AnalysisPage: React.FC = () => {
             expandedViolations={expandedViolations}
             setExpandedViolations={setExpandedViolations}
           />
+        ) : !serverRunning ? (
+          <EmptyState>
+            <Title headingLevel="h2" size="lg">
+              Server Not Running
+            </Title>
+            <EmptyStateBody>
+              The analysis server is not running. Please start the server to run an analysis.
```

Comment:
```suggestion
              The server is not running. Please start the server to run an analysis.
```

Since we will be starting kai and not the analyzer...think we may be better off referring to it as just "server".